tests/bats/in_container/bats_utils.bash

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+AIRFLOW_SOURCES=$(pwd)
+export AIRFLOW_SOURCES
+readonly AIRFLOW_SOURCES
+
+export DOCKER_BINARY_PATH=${AIRFLOW_SOURCES}/tests/bats/mock/docker.sh
```

Comment:
All those lines are not really needed. They are needed for Host scripts (in the host scripts we are loading _all_libs.sh and we are executing al the kubectl/kind etc .- but inside the container those are not really needed. I think the whole load bat_utls. step is not needed in fact. Maybe we will come up with some common utils to load over time but I think no need for this now.

Reply:
Thanks, fixed it 35c2be2